app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/finance/MoneyUtils.scala

```diff
@@ -56,9 +56,13 @@ object MoneyUtils {
 
   def validateValueIsLessThan(key: String, limitInclusive: AmountInPence, errorMessageKey: => String)(
     value: BigDecimal
-  ): Either[FormError, BigDecimal] =
+  )(implicit messages: Messages): Either[FormError, BigDecimal] =
```

Comment:
is this what was breaking it? What does this do here?

Reply:
Nope, removed.